helpers.go

```diff
@@ -140,5 +140,9 @@ func quote(v driver.Value) string {
 }
 
 func formatTime(v time.Time) string {
```

Comment:
@gingerwizard I don't understand what it is looking at the diff.
What is `2006-01-02 15:04:05`? Would this change affect performance?

Need your help.

Reply:
So if the user sets the timezone to Local in their insert e.g. 

```connect.QueryRow("SELECT COUNT(*) FROM clickhouse_test_select WHERE datetime = ?", time.Date(2017, 1, 20, 14, 0, 0, 0, time.Local))```

Currently we serialized "Local" into the string sent to ClickHouse - wrapped in the toDateTime function. Proposal here is to send epoch instead.
Few options:

1. Send epoch at this PR does
2. Find the timezone t.Zone() and do the current strategy - i.e. dyanmic sys lookup. Likely slower.

I wonder what is most efficient for ClickHouse? @alexey-milovidov  i.e. why not just always send epoch to CH?

Note this is v1 and tests for this are failing - question remains re support for v1.

Reply:
Well, I did try to mimic what v2 does (see https://github.com/ClickHouse/clickhouse-go/issues/535)

Reply:
Yeh trying to figure out the authors original intention here..this is for queries..unclear as to the benefit of sending as a string and not just a unit32 - the former CH has to the conversion. Checking with core.

Reply:
This fixes the issue and v1 is patches only at this point.